tools/kompos/index.html

```diff
@@ -211,14 +221,13 @@
     padding: 0.1rem .2rem;
     overflow: hidden;
   }
-
   .activity-scopes {
     width: 100%;
     margin-top: 2ex;
   }
   .activity-scopes th, .activity-scopes td {
     padding: 0.1rem .2rem;
-    font-size: 9px
+    font-size: 12px
```

Comment:
please don't change these visual things as part of this pull request.
They have nothing to do with breakpoints.